<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,60,20,5">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QLineEdit" name="search_bar">
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_movie">
       <attribute name="title">
        <string>Movies</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="movie_tree">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Length</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Plugin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Rating</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_tv">
       <attribute name="title">
        <string>TV Shows</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="tv_tree">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Length</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Seasons</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Episodes</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Plugin</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quality</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="poster">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>190</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#poster { padding: 3px; background: #d2d2d2}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="10,60,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="movie_title">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>No Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="description">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#description {
	border: 1px solid #d2d2d2;
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;No movie description is availabe at this time. Sorry.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="40,40,10">
          <item>
           <widget class="QComboBox" name="quality_combo">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="subtitle_combo"/>
          </item>
          <item>
           <widget class="QPushButton" name="play_button">
            <property name="text">
             <string>PLAY!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="status_bar">
      <property name="text">
       <string>READY</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPrefences">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionVideo_player"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menuPrefences"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionVideo_player">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
